.github/workflows/build_commits.yaml

```diff
@@ -0,0 +1,51 @@
+name: build_commits
+
+on: [pull_request]
+
+jobs:
+  build_commits:
+    name: build commit code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@37335c7bb261b353407cff977110895fa0b4f7d8
+        with:
+          go-version: 1.16.3
+          
+      - name: Cache LLVM and Clang
+        id: cache-llvm
+        uses: actions/cache@v2.1.4
```

Comment:
Sorry, I wasn't clear in [my previous comment](https://github.com/cilium/cilium/pull/15659#discussion_r612350777): we should pin all actions to a SHA, and not only the Go one which was provided as an example ;)

Reply:
@nbusseneau I tried to find the SHA for the other tags but couldn't find it. Can you please point to the resource where I could find the SHA for the rest of the tags?

Reply:
Of course! Don't hesitate to ask :) Here is what I usually do:

- Copy/paste the action name on Google (for example, here `actions/cache`)
- Find the GitHub repository for this GitHub actions, either directly from Google or from the GitHub Actions marketplace page (here it would be https://github.com/actions/cache)
- Go to the "Releases" page and click on the SHA associated with the latest version (or the version I want to use, if specific) (here it would be the `1a9e213` link)
- Retrieve the full SHA from the page or URL (here it would be `1a9e2138d905efd099035b49d8b7a3888c653ca8`)

Reply:
@nbusseneau I tried to find the SHA for the other tags but couldn't find them. Can you please point to the resource where I could find the SHA for the rest of the tags?